src/components/plots/stackedAreaPlot.ts

```diff
@@ -69,6 +69,13 @@ export module Plots {
       return this;
     }
 
+    protected _updateExtentsForAttr(attr: string) {
+      super._updateExtentsForAttr(attr);
+      if ((attr === "x" || attr === "y") && this._projectorsReady()) {
+        this._updateStackExtents();
+      }
```

Comment:
Yes but now we have duplication of logic when they should derive from a single point of truth